eureka_ml_insights/user_configs/arc_agi.py

```diff
@@ -0,0 +1,347 @@
+import os
+from typing import Any
+
+from eureka_ml_insights.core import Inference, PromptProcessing
+from eureka_ml_insights.core.data_processing import DataProcessing
+from eureka_ml_insights.core.eval_reporting import EvalReporting
+from eureka_ml_insights.data_utils.arc_agi_utils import (
+    ARCAGI_ExtractAnswer
+)
+from eureka_ml_insights.data_utils.data import (
+    DataLoader,
+    DataReader,
+    HFDataReader,
+)
+from eureka_ml_insights.metrics.metrics_base import ExactMatch
+from eureka_ml_insights.metrics.reports import (
+    CountAggregator,
+    AverageAggregator,
+    BiLevelCountAggregator,
+    BiLevelAggregator,
+    CountAggregator
+)
+
+from eureka_ml_insights.data_utils.transform import (
+    AddColumn,
+    AddColumnAndData,
+    CleanCOTAnswer,
+    ColumnRename,
+    CopyColumn,
+    ExtractUsageTransform,
+    MajorityVoteTransform,
```

Comment:
it looks like there are unused imports here. please run our formatting commands to clean up the code. See "how to contribute" in readme. TLDR: make format-inplace